icechunk-python/src/repository.rs

```diff
@@ -591,10 +591,7 @@ impl PyRepository {
     }
 
     #[staticmethod]
-    fn exists_async<'py>(
-        py: Python<'py>,
-        storage: PyStorage,
```

Comment:
changes in the file only because pre-commit was bitterly complaining if i didn't also change this. No functionality changes here as far as I can tell

Reply:
you may have a too old/too new Rust version